src/servers/src/heartbeat_options.rs

```diff
@@ -0,0 +1,17 @@
+use serde::{Deserialize, Serialize};
```

Comment:
@Kree0 could you please add the licence header here? you can copy it from any existing file